rpc/src/server/router.rs

```diff
@@ -38,6 +38,7 @@ pub(crate) fn create_routes() -> PathTree<Handler> {
     routes.handle("/stats/memory", dev_handler::dev_stats_memory);
     routes.handle("/p2p/:offset/:count", dev_handler::p2p_messages);
     routes.handle("/p2p/:offset/:count/:host", dev_handler::p2p_host_messages);
+    //routes.handle("/stats/storage", dev_handler::dev_stats_storage);
```

Comment:
why is this commented?

Reply:
Juraj asked me to comment it out for the time being. It's a very resource-intense endpoint and if someone starts spamming it they could bring down the server.